framework/components/ecl-dialogs/ecl-dialogs.scss

```diff
@@ -0,0 +1,45 @@
+/**
+ * Dialog
+ * @define dialog
+ */
+
+.ecl-dialog {
+  background-color: #fff;
+  left: 50%;
+  margin: 0;
+  padding: map-get($ecl-spacing, m);
+  position: fixed;
+  text-align: center;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 90%;
+  z-index: map-get($ecl-z-index, 'modal');
+
+  &[aria-hidden='true'],
```

Comment:
I think it'd be even better to reuse the global reset approach. Thanks for pointing out a possible improvement!